core/templates/css/oppia.css

```diff
@@ -1674,6 +1674,7 @@ pre.oppia-pre-wrapped-text {
 
 .oppia-editor-card-section {
   padding: 20px 50px 20px 35px;
+  word-break: break-word;
```

Comment:
Why is this required?

Reply:
@kevintab95, I noticed a display bug while saving the interaction, If the input is a large strings with no breaks the text overflows. 

For example:
![Screenshot from 2020-12-17 13-27-25](https://user-images.githubusercontent.com/66058206/102509706-dbbbc280-40ac-11eb-9f45-67b8d8bdc022.png)

Reply:
Ah ok so this rule should fix it. Cool, thanks!